app/articles/tests/test_views.py

```diff
@@ -108,3 +108,145 @@ def test_delete_article_without_authentication(self) -> None:
             json.loads(response.content).get("detail"),
             "Authentication credentials were not provided.",
         )
+
+
+class TestBookmarkView(TestCase):
+    """
+    Test for article bookmarks
+    """
+
+    password: str
+    user: Any
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        super().setUpClass()
+        cls.password = fake.password()
+        cls.user = User.objects.create_user(
+            username=fake.name(), email=fake.email(), password=cls.password
+        )
+
+    @property
+    def bearer_token(self) -> dict:
+        login_url = reverse("login")
+        response = self.client.post(
+            login_url,
+            data={"email": self.user.email, "password": self.password},
+        )
+        token = json.loads(response.content).get("access")
+        return {"HTTP_AUTHORIZATION": f"Bearer {token}"}
+
+    def test_bookmark_article(self) -> None:
+        """
+        Test if a user can bookmark an article
+        """
+        data = Article.objects.create(
+            post_id="9831040d-5eeb-4cd5-9054-d0f1770431ca",
```

Comment:
stop manually putting the id

Reply:
I tried calling it automatically it keeps giving me an assertion error.